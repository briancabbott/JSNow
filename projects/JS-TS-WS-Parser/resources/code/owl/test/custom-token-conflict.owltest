other other
---
.token other
input = other 'other'
