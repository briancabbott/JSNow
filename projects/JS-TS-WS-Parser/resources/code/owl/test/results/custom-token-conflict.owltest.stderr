error: this user-defined token can't share its name with a keyword

  .token other
         ~~~~~
  input = other 'other'
                ~~~~~~~

